SimTracker/TrackerMaterialAnalysis/test/listIds_PhaseII.py

```diff
@@ -4,16 +4,7 @@
 
 process = cms.Process("MaterialAnalyser")
 
-readGeometryFromDB = False
-
-if not readGeometryFromDB:
-  process.load('Configuration.Geometry.GeometryExtended2026D41Reco_cff')
-else:
-# GlobalTag and geometry via GT
-  process.load('Configuration.Geometry.GeometrySimDB_cff')
-  process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
```

Comment:
since you are at it, can you clean up the `condDBv2` here and in all the other files in this package? 
That's now unnecessary.

Reply:
See comment above regarding reading from DB

Reply:
In the modified files, yes.